libr/core/cmd_open.c

```diff
@@ -570,10 +571,17 @@ static void cmd_open_map(RCore *core, const char *input) {
 	case '.': // "om."
 		map = r_io_map_get (core->io, core->offset);
 		if (map) {
-			core->print->cb_printf ("map: %i fd: %i +0x%"PFMT64x" 0x%"PFMT64x
-				" - 0x%"PFMT64x" ; %s : %s\n", map->id, map->fd,
-				map->delta, map->itv.addr, r_itv_end (map->itv),
-			r_str_rwx_i (map->perm), map->name ? map->name : "");
+			if (input[2] == 'j') { // "om.j"
+				core->print->cb_printf ("{\"map\":%2d,\"fd\":%d,\"delta\":%"PFMT64u",\"from\":%"PFMT64u
```

Comment:
please use the PJ api to construct the JSON output

Reply:
which PJ api?

Reply:
the only pj api we have (libr/util/pj.c) see 046a1fe32474246e982e3bab313028f9631917b4 as an example